composer.json

```diff
@@ -6,11 +6,15 @@
     "license": "MIT",
     "require": {
         "cakephp/console": "^4.0",
-        "rector/rector": "^0.14.5"
+        "nette/utils": "^3.2",
+        "rector/rector": "dev-main",
+        "symfony/string": "^6.0",
```

Comment:
this is needed as use `Symfony\Component\String\UnicodeString` in ChangeSnakedFixtureNameToPascalRector rule.